### DISCUSSION TITLE
How to specify values for 'axes' property in `next/font/google` for variable fonts?

### DISCUSSION THREAD

**Discussion by BaileySimrell** [vercel/next.js#64959] [Help]
### Summary

I'm working on integrating the `Bricolage_Grotesque` variable font into my Next.js application using `next/font/google`. However, I'm having trouble understanding how to set specific values for the font's axes, such as width ('wdth'). The type definitions seem to indicate that `axes` is an array, but there's no clear way to assign values to each axis.

### Additional information

```js
// This does not specify a value for 'wdth'
export const bricolage = Bricolage_Grotesque({
  subsets: ['latin'],
  axes: ['wdth']
});

// Expected usage (or similar), which does not work:
export const bricolage = Bricolage_Grotesque({
  subsets: ['latin'],
  axes: {
    'wdth': 80.7
  }
});

```

- Next.js version: 14.1.3
- @next/font version: 14.2.2
- I checked the official Next.js documentation for guidance on this feature but couldn't find any examples or clear instructions on using the axes property.
- I attempted various syntaxes to assign a value to the 'wdth' axis, but none have compiled without errors.

### The following will compile, but I don't know how to change the actual width value:
<img width="612" alt="Screenshot 2024-04-24 at 4 25 41 AM" src="https://github.com/vercel/next.js/assets/6186817/a1ba9c38-401c-4b41-8ed6-21dec195943d">

### Things I tried that didn't work:
<img width="595" alt="Screenshot 2024-04-24 at 3 55 44 AM" src="https://github.com/vercel/next.js/assets/6186817/8b40bdab-152e-47b2-9fb7-49dad666a91a">
<img width="439" alt="Screenshot 2024-04-24 at 4 04 03 AM" src="https://github.com/vercel/next.js/assets/6186817/e50f88f5-35d7-4bca-bacb-ec2847a2eacc">
<img width="510" alt="Screenshot 2024-04-24 at 4 03 57 AM" src="https://github.com/vercel/next.js/assets/6186817/4705c80a-9e82-4ff3-b659-c983a60f49bc">
<img width="338" alt="Screenshot 2024-04-24 at 3 58 53 AM" src="https://github.com/vercel/next.js/assets/6186817/2390b82a-8245-4589-89fc-4387cb7c4541">
<img width="605" alt="Screenshot 2024-04-24 at 3 55 51 AM" src="https://github.com/vercel/next.js/assets/6186817/db8b2e5d-823f-4059-8e60-d5aac1b1afdf">

### Another random thing I tried: 
I also tried seeing if the `wdth` axes value would be configurable on the exported object, but that didn't work or lead me to anything... Based on the type definition of `Bricolage_Grotesque`, I am pretty confused on this one tbh.

<img width="496" alt="Screenshot 2024-04-24 at 4 10 23 AM" src="https://github.com/vercel/next.js/assets/6186817/0dd9f4f8-92fc-4a92-939d-3289c6861c70">

**Comment by millarbart** [comment]
I was able to get it to work with Roboto-Flex by using:
  axes: ['wdth']

then in the css using the property:
font-stretch: 62%;
